var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -33,41 +33,43 @@ class SuiteSparse(Package):
     url = 'http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.1.tar.gz'
 
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
+    version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
 
     # FIXME: (see below)
     # variant('tbb', default=True, description='Build with Intel TBB')
 
     depends_on('blas')
     depends_on('lapack')
 
-    depends_on('metis@5.1.0', when='@4.5.1')
+    depends_on('metis@5.1.0', when='@4.5.1:')
```

Comment:
i think they have somewhere an officially recommended metis for each suite-sparse release.